docs/antctl.md

```diff
@@ -781,6 +781,34 @@ PEER               ASN   STATE
 192.168.77.201:179 65002 Active
 ```
 
+`antctl` agent command `get bgproutes` prints the advertised BGP routes.
+IPv4 routes are advertised to all IPv4 BGP peers. IPv6 routes are advertised to all IPv6 BGP peers.
+
+```bash
+# Get the list of advertised IPv4 bgp routes
+$ antctl get bgproutes --ipv4
```

Comment:
why do we want to provide the ability to filter based on the IP family for this command (`get bgproutes`), while we don't provide the same ability for `get bgppeers`?

Reply:
We can add the same ability for `get bgppeers` as well. I didn't thought about this ability then.
I thought about this ability after reading this comment https://github.com/antrea-io/antrea/pull/6734#discussion_r1797534858 . I don't think there is any harm adding this ability.
Do you think we should not keep it ?

Reply:
I think it is ok to keep it. I am not convinced it is very useful but I do not see the harm in having it either.